packages/button/src/styles.ts

```diff
@@ -27,6 +27,12 @@ const baseButtonStyles = css`
   &:disabled {
     pointer-events: none;
   }
+
+  &:active,
+  &:focus,
```

Comment:
We need to also set the color explicitly for focus outside of our leafygreen provider handling